src/config.c

```diff
@@ -862,6 +934,50 @@ void RSConfigExternalTrigger_Register(RSConfigExternalTrigger trigger, const cha
   RSGlobalConfigTriggers[numTriggers++] = trigger;
 }
 
+#ifdef MT_BUILD
+
+void UpgradeDeprecatedMTConfigs() {
+  RSConfigVar *workers = findConfigVar(&RSGlobalConfigOptions, "WORKERS");
+  RSConfigVar *minOperationWorkers = findConfigVar(&RSGlobalConfigOptions, "MIN_OPERATION_WORKERS");
+  if (workers->flags & RSCONFIGVAR_F_MODIFIED || minOperationWorkers->flags & RSCONFIGVAR_F_MODIFIED) {
```

Comment:
What if only one of them was set? should we still ignore the deprecated ones?

Reply:
Yes. IMO any usage of the new configs proves the user has adapted to the new configurations. If we don’t make a clear cut we will face all kinds of weird edge cases

Reply:
The user should not load the module with both old and new configurations. The goal here is to avoid ambiguities, not to create new ones.
We support loading the module exclusively with either the old or the new configurations. If we get both, we ignore the old ones.

Reply:
Support for unambiguous combination of old and new configurations was added